### PR TITLE
Get cloud provider resources

### PR DESCRIPTION
**Pull request by YiscahLevySilas1**
## Overview
Get new cloud provider resource - DescribeRepositories. 
## Additional Information 
DescribeRepositories is a resource that will be used in upcoming implementations of CIS benchmarks in regolibrary.
## How to Test
Once CIS-EKS benchmark will be release, can be tested by running: kubescape scan control C-0221 (or: kubescape scan framework cis-eks-t1.2.0)

## Checklist before requesting a review

put an [x] in the box to get it checked 

- [x] My code follows the style guidelines of this project
- [x] I have commented on my code, particularly in hard-to-understand areas
- [x] I have performed a self-review of my code
- [x] If it is a core feature, I have added thorough tests.
- [x] New and existing unit tests pass locally with my changes

**Please open the PR against the `dev` branch (Unless the PR contains only documentation changes)**

-->

### CODE REVIEW

**Comment by dwertent** [comment]
So this is related to the `CIS-EKS` epic.
Will we need to develop anything else related to the `CIS-EKS` story? If so, we might want to create a "feature branch" that will be dedicated to this purpose. Then, we will merge the feature branch once we complete all related development.

**Comment by YiscahLevySilas1** [comment]
@dwertent Yes, we probably will.
Sure I'll open a separate branch for now

File: core/pkg/resourcehandler/k8sresources.go
Comment (kooomix) [inline core/pkg/resourcehandler/k8sresources.go:175]: This check can be done within getCloudProviderDescriptionRepositories function. Within the function, you can run `cloudResources := cautils.MapCloudResources(ksResourceMap)` to get the cloudResources.

**Review by kooomix — CHANGES_REQUESTED** [review]

File: core/pkg/resourcehandler/k8sresources.go
Comment (YiscahLevySilas1) [reply core/pkg/resourcehandler/k8sresources.go:175]: I already have the cloudResources at this stage, I want to check here if they include a specific resource

Comment (kooomix) [reply core/pkg/resourcehandler/k8sresources.go:175]: I think it will be cleaner to have this check within the function, but your call.

**Review by kooomix — APPROVED** [review]

**Review by kooomix — APPROVED** [review]

**Pull request merged**
